core/dom/range.js

```diff
@@ -2849,11 +2849,12 @@ CKEDITOR.dom.range = function( root ) {
 		 * <img src="https://33333.cdn.cke-cs.com/rc1DFuFpHqcR3Mah6y0e/images/90893fcc6c323c10023e73ebfc1fbaa622b48b29c066f7af_ie-rects.png">
 		 *
 		 * @since 4.10.0
+		 * @param {Boolean} [isAbsolute=false] The function will retrieve an absolute rectangle of the element i.e. position relative to the upper-left corner of the topmost viewport.
```

Comment:
Please split the line into two, try not exceed 160 chars in a line.

Reply:
Ok, but shouldn't linters check for that value?